src/pages/dashboard/index.page.tsx

```diff
@@ -0,0 +1,216 @@
+import { useT } from "@transifex/react";
+import { useContext } from "react";
+
+import Text from "@/components/elements/Text/Text";
+import ToolTip from "@/components/elements/Tooltip/Tooltip";
+import Icon, { IconNames } from "@/components/extensive/Icon/Icon";
+import PageCard from "@/components/extensive/PageElements/Card/PageCard";
+import PageRow from "@/components/extensive/PageElements/Row/PageRow";
+
+import ContentOverview from "./components/ContentOverview";
+import SecDashboard from "./components/SecDashboard";
+import { RefContext } from "./context/ScrollContext.provider";
+import {
+  JOBS_CREATED_BY_AGE,
+  JOBS_CREATED_BY_GENDER,
+  LABEL_LEGEND,
+  NEW_FULL_TIME_JOBS,
+  NEW_PART_TIME_JOBS,
+  NUMBER_OF_TREES_PLANTED,
+  NUMBER_OF_TREES_PLANTED_BY_YEAR,
+  RESTORATION_STRATEGIES_REPRESENTED,
+  TARGET_LAND_USE_TYPES_REPRESENTED,
+  TOP_10_PROJECTS_WITH_THE_MOST_PLANTED_TREES,
+  TOTAL_HECTARES_UNDER_RESTORATION,
+  TOTAL_NUMBER_OF_SITES,
+  TOTAL_VOLUNTEERS,
+  VOLUNTEERS_CREATED_BY_AGE,
+  VOLUNTEERS_CREATED_BY_GENDER
+} from "./mockedData/dashboard";
+
+export interface DashboardTableDataProps {
+  label: string;
+  valueText: string;
+  value: number;
+}
+
+export interface GraphicLegendProps {
+  label: string;
+  value: string;
+  color: string;
+}
+
+export interface DashboardDataProps {
+  value?: string;
+  unit?: string;
+  secondValue?: string;
+  graphic?: string;
+  tableData?: DashboardTableDataProps[];
+  maxValue?: number;
+  graphicLegend?: GraphicLegendProps[];
+  graphicTargetLandUseTypes?: DashboardTableDataProps[];
+}
+
+const Dashboard = () => {
+  const t = useT();
+  const dataToggle = ["Absolute", "Relative"];
+  const dataToggleGraphic = ["Table", "Graphic"];
+  const sharedRef = useContext(RefContext);
+  const dashboardHeader = [
+    {
+      label: "Trees Planted",
+      value: "12.2M"
+    },
+    {
+      label: "Hectares Under Restoration",
+      value: "5,220 ha"
+    },
+    {
+      label: "Jobs Created",
+      value: "23,000"
+    }
+  ];
+
+  return (
+    <div className="flex flex-1 gap-4 overflow-hidden bg-neutral-70 p-4 ">
+      <div ref={sharedRef} className="w-3/5 overflow-auto pr-2 ">
+        <PageRow className="gap-4 p-0">
+          <div className="grid w-full grid-cols-3 gap-4">
+            {dashboardHeader.map((item, index) => (
+              <div key={index} className="rounded-lg bg-white px-5 py-4.5">
+                <Text variant="text-10-light" className="text-darkCustom opacity-60">
+                  {item.label}
```

Comment:
This labels should be translated, use Transifex

Reply:
done